### PR TITLE
Add licenses to sub packages

### PR DESCRIPTION
**Pull request by honzajavorek**
Fix https://github.com/airbnb/javascript/issues/1744. Not sure how to trigger patch release, couldn't find any contributing docs. Please advise.

Replaces #1745

### CODE REVIEW

File: packages/eslint-config-airbnb-base/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2012 Airbnb
```
Comment (raunofreiberg) [inline packages/eslint-config-airbnb-base/LICENSE.md]: 2012?

File: packages/eslint-config-airbnb/LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2012 Airbnb
```
Comment (raunofreiberg) [inline packages/eslint-config-airbnb/LICENSE.md]: Ditto.

**Comment by honzajavorek** [comment]
@ljharb Thank you! While I'm no lawyer myself, I think although the package is a product of the repository with a license text, it is still necessary to accompany the package with a full license text, as it is distributed separately. And the MIT license requires the full license text to be alongside the code. Will link this as well: https://github.com/babel/babel/pull/7308#issuecomment-367615152

**Comment by honzajavorek** [comment]
@ljharb btw, do you plan a new release?

**Comment by ljharb** [comment]
The next release will be breaking (due to other merged changes); I’m not going to backport a patch release just for this.

**Comment by ljharb** [comment]
Separately, the point of a SPDX-compliant identifier, as i understand it, is that the license text is no longer relevant or needed because it is uniquely identified by the license identifier (which implies license text). Maybe @kemitchell can confirm, or correct me if I’m wrong?

**Comment by honzajavorek** [comment]
> I’m not going to backport a patch release just for this.

I didn't think of it like that. It was more like whether you have an idea when the next release might happen.

**Comment by ljharb** [comment]
No idea, but hopefully soon.

**Pull request merged**